### ISSUE TITLE
Working at a popular company!

### ISSUE THREAD

**Issue by siddharthkp** [kentcdodds/ama#312]
Hey!

How has working for a famous + big company like PayPal affected your career.

Would you say that the big size limits the impact you can make?
Would you say that your personal work gets amplified because of the brand name?

And lastly, would you say there is a right time to pick a famous company over a smaller one in the context of your answers above?

**Comment by kentcdodds** [comment]
Hey @siddharthkp!

So working at PayPal has been super! I haven't really had a chance to leverage the "resume building" that working at a big company like PayPal has done, but it's nice to be able to say where I work and not have to explain what my company does 😅 It's also awesome because I'm part of a company that has a major impact on the world. People use PayPal to support their families and help their friends (I work on Peer-To-Peer payments). That's pretty cool.

My company before PayPal was Alianza (see #1 for more on that story). It's a small company in Utah that I really enjoyed working at. I was hired on to replace one frontend engineer. The rest of the engineers were backend. I was responsible for the entire frontend. It was an incredibly awesome learning experience. I had to learn _everything_ about building, testing, and deploying a frontend application. Wouldn't replace the experience for anything.

I did eventually leave in part because I wanted to have more engineers to rub shoulders with and also I wanted to start doing React and that wasn't an option for Alianza.

To answer your questions...

> Would you say that the big size limits the impact you can make?

Well, my impact on the company is a little limited due to the size of the company. However, my impact on the world is much bigger than it was at Alianza. Way more people use my stuff than they did when I worked at Alianza :wink:

> Would you say that your personal work gets amplified because of the brand name?

Probably... But it's hard for me to quantify that...

> And lastly, would you say there is a right time to pick a famous company over a smaller one in the context of your answers above?

I would say that working at a smaller company early in my career was a pretty good move. But it wasn't my first company. I had about a year with a few other companies (internships etc.) where I learned a bunch of foundational stuff. I think that's important. I'm the kind of developer that likes to have far-reaching responsibilities. I like working to make developers more efficient/effective/etc. That's the kind of work I'm doing at PayPal now and I don't think that I'd be quite ready for it if I hadn't worked at Alianza first.

I hope that's helpful! Good luck!
